atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/sweep/AbstractSweepTaskRunnerTest.java

```diff
@@ -488,7 +495,9 @@ public void testSweepBatchesUpToDeleteBatchSize() {
 
         putTwoValuesInEachCell(SMALL_LIST_OF_CELLS);
 
-        List<List<Cell>> sweptCells = runSweep(cellsSweeper, spiedSweepRunner, 8, 1, 4);
+        Pair<List<List<Cell>>, SweepResults> sweptCellsAndSweepResults =
```

Comment:
nit: would be nice to have "swept cells and sweep results" encapsulated as a private class, but I can't immediately think of a good name for it.